[2022-12-27T03:23:46.920+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: final_project_dag.extract_test manual__2022-12-27T03:17:40.630548+00:00 [queued]>
[2022-12-27T03:23:47.020+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: final_project_dag.extract_test manual__2022-12-27T03:17:40.630548+00:00 [queued]>
[2022-12-27T03:23:47.031+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-27T03:23:47.059+0000] {taskinstance.py:1284} INFO - Starting attempt 2 of 6
[2022-12-27T03:23:47.069+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-27T03:23:47.196+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): extract_test> on 2022-12-27 03:17:40.630548+00:00
[2022-12-27T03:23:47.257+0000] {standard_task_runner.py:55} INFO - Started process 490 to run task
[2022-12-27T03:23:47.286+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'final_project_dag', 'extract_test', 'manual__2022-12-27T03:17:40.630548+00:00', '--job-id', '299', '--raw', '--subdir', 'DAGS_FOLDER/final_project_dag.py', '--cfg-path', '/tmp/tmpg1hznm7t']
[2022-12-27T03:23:47.303+0000] {standard_task_runner.py:83} INFO - Job 299: Subtask extract_test
[2022-12-27T03:23:51.158+0000] {task_command.py:389} INFO - Running <TaskInstance: final_project_dag.extract_test manual__2022-12-27T03:17:40.630548+00:00 [running]> on host b40088984618
[2022-12-27T03:23:52.218+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kelompok 3
AIRFLOW_CTX_DAG_ID=final_project_dag
AIRFLOW_CTX_TASK_ID=extract_test
AIRFLOW_CTX_EXECUTION_DATE=2022-12-27T03:17:40.630548+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-27T03:17:40.630548+00:00
[2022-12-27T03:23:52.448+0000] {conn.py:372} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:9092 <connecting> [IPv4 ('172.25.0.4', 9092)]>: connecting to broker:9092 [('172.25.0.4', 9092) IPv4]
[2022-12-27T03:23:52.459+0000] {conn.py:1049} INFO - Probing node bootstrap-0 broker version
[2022-12-27T03:23:52.467+0000] {conn.py:401} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:9092 <connecting> [IPv4 ('172.25.0.4', 9092)]>: Connection complete.
[2022-12-27T03:23:52.820+0000] {conn.py:1106} INFO - Broker version identifed as 1.0.0
[2022-12-27T03:23:52.827+0000] {conn.py:1108} INFO - Set configuration api_version=(1, 0, 0) to skip auto check_version requests on startup
[2022-12-27T03:23:52.880+0000] {subscription_state.py:171} INFO - Updating subscribed topics to: ('projek_akhir',)
[2022-12-27T03:23:53.944+0000] {streaming.py:217} ERROR - Stream encountered HTTP error: 429
[2022-12-27T03:23:53.951+0000] {streaming.py:111} ERROR - HTTP error response text: {"title":"ConnectionException","detail":"This stream is currently at the maximum allowed connection limit.","connection_issue":"TooManyConnections","type":"https://api.twitter.com/2/problems/streaming-connection"}
[2022-12-27T03:24:54.583+0000] {logging_mixin.py:137} INFO - Connected
[2022-12-27T03:24:54.602+0000] {streaming.py:185} INFO - Stream connected
[2022-12-27T03:24:58.435+0000] {local_task_job.py:224} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2022-12-27T03:24:58.459+0000] {process_utils.py:133} INFO - Sending Signals.SIGTERM to group 490. PIDs of all processes in the group: [490]
[2022-12-27T03:24:58.469+0000] {process_utils.py:84} INFO - Sending the signal Signals.SIGTERM to group 490
[2022-12-27T03:24:58.487+0000] {taskinstance.py:1483} ERROR - Received SIGTERM. Terminating subprocesses.
[2022-12-27T03:24:58.505+0000] {streaming.py:203} ERROR - Stream encountered an exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/tweepy/streaming.py", line 91, in _connect
    chunk_size=self.chunk_size
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/models.py", line 866, in iter_lines
    chunk_size=chunk_size, decode_unicode=decode_unicode
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/models.py", line 816, in generate
    yield from self.raw.stream(chunk_size, decode_content=True)
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/response.py", line 624, in stream
    for line in self.read_chunked(amt, decode_content=decode_content):
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/response.py", line 828, in read_chunked
    self._update_chunk_length()
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/response.py", line 758, in _update_chunk_length
    line = self._fp.fp.readline()
  File "/usr/local/lib/python3.7/socket.py", line 589, in readinto
    return self._sock.recv_into(b)
  File "/usr/local/lib/python3.7/ssl.py", line 1071, in recv_into
    return self.read(nbytes, buffer)
  File "/usr/local/lib/python3.7/ssl.py", line 929, in read
    return self._sslobj.read(len, buffer)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1485, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2022-12-27T03:24:58.588+0000] {logging_mixin.py:137} INFO - Disconnected
[2022-12-27T03:24:58.595+0000] {streaming.py:193} INFO - Stream disconnected
[2022-12-27T03:24:58.619+0000] {conn.py:372} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:9092 <connecting> [IPv4 ('172.25.0.4', 9092)]>: connecting to broker:9092 [('172.25.0.4', 9092) IPv4]
[2022-12-27T03:24:58.735+0000] {conn.py:401} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:9092 <connecting> [IPv4 ('172.25.0.4', 9092)]>: Connection complete.
[2022-12-27T03:24:58.886+0000] {cluster.py:376} INFO - Group coordinator for consumers is (1003, 'broker', 9092, None)
[2022-12-27T03:24:58.897+0000] {base.py:688} INFO - Discovered coordinator 1003 for group consumers
[2022-12-27T03:24:58.915+0000] {base.py:735} INFO - Starting new heartbeat thread
[2022-12-27T03:24:58.936+0000] {consumer.py:342} INFO - Revoking previously assigned partitions () for group consumers
[2022-12-27T03:24:58.950+0000] {conn.py:372} INFO - <BrokerConnection node_id=1003 host=broker:9092 <connecting> [IPv4 ('172.25.0.4', 9092)]>: connecting to broker:9092 [('172.25.0.4', 9092) IPv4]
[2022-12-27T03:24:59.068+0000] {conn.py:401} INFO - <BrokerConnection node_id=1003 host=broker:9092 <connecting> [IPv4 ('172.25.0.4', 9092)]>: Connection complete.
[2022-12-27T03:24:59.081+0000] {conn.py:811} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:9092 <connected> [IPv4 ('172.25.0.4', 9092)]>: Closing connection. 
[2022-12-27T03:24:59.205+0000] {base.py:447} INFO - (Re-)joining group consumers
[2022-12-27T03:25:00.070+0000] {base.py:516} INFO - Elected group leader -- performing partition assignments using range
[2022-12-27T03:25:00.149+0000] {base.py:333} INFO - Successfully joined group consumers with generation 2
[2022-12-27T03:25:00.152+0000] {subscription_state.py:257} INFO - Updated partition assignment: [('projek_akhir', 0)]
[2022-12-27T03:25:00.155+0000] {consumer.py:239} INFO - Setting newly assigned partitions (('projek_akhir', 0),) for group consumers
[2022-12-27T03:25:00.612+0000] {logging_mixin.py:137} INFO - <bound method NDFrame.head of Empty DataFrame
Columns: [tweet_id, text, user_id, created_at, source]
Index: []>
[2022-12-27T03:25:00.616+0000] {base.py:742} INFO - Stopping heartbeat thread
[2022-12-27T03:25:00.629+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-12-27T03:25:00.782+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=final_project_dag, task_id=extract_test, execution_date=20221227T031740, start_date=20221227T032346, end_date=20221227T032500
[2022-12-27T03:25:01.651+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=490, status='terminated', exitcode=0, started='03:23:47') (490) terminated with exit code 0
